client/src/components/forms/Login.js

```diff
@@ -1,16 +1,23 @@
 import React, { useState } from "react";
 import styled from "styled-components";
-import { Link } from "react-router-dom";
+import { Link, useHistory } from "react-router-dom";
 import { postData } from "../../helpers/fetch";
 
-function Login() {
+function Login({ setUserData, userData }) {
+  console.log(setUserData);
   const [input, setInput] = useState({});
+  const history = useHistory();
 
-  const handleSubmit = () => {
-    fetch("/login", {
+  const handleSubmit = async () => {
+    const apiCall = await fetch("/login", {
       ...postData,
       body: JSON.stringify(input),
     });
+    if (apiCall.status === 200) {
```

Comment:
Also, handle cases where the status isn't 200. Don't leave the user wondering what happened 😢 . You could display a helpful error message.